projects/packages/forms/src/contact-form/class-contact-form.php

```diff
@@ -1261,10 +1261,22 @@ public static function delete_feedback_files( $post_id ) {
 	 * The output HTML will have a few extra escapes, but that makes no functional difference.
 	 *
 	 * @since 9.1.0
-	 * @param string $val Value to escape.
+	 * @param string|array $val Value to escape.
 	 * @return string
 	 */
 	public static function esc_shortcode_val( $val ) {
```

Comment:
It would be good to add a test for this change. So it is a bit more clear how this change impacts things

Reply:
Indeed, I'll see to it. The problem with the `string` assumption is that we get, from time to time, that PHP warning about array-to-string conversion. I only added the recursivity chunk so the method would call itself.